builder.go

```diff
@@ -46,7 +48,7 @@ func (b *Builder) Build() (string, error) {
 	}
 
 	for _, v := range b.locals {
-		s += ".local" + b.spacing + string(v.variable) + b.spacing + "=" + b.spacing + v.expr.build(b.spacing) + b.newline
+		s += ".local" + b.spacing + varSymbol + string(v.variable) + b.spacing + "=" + b.spacing + v.expr.build(b.spacing) + b.newline //nolint:lll
```

Comment:
It's ok to break this line

```suggestion
		s += ".local" + b.spacing + varSymbol + string(v.variable) + b.spacing + "=" +
		b.spacing + v.expr.build(b.spacing) + b.newline
```